### ISSUE TITLE
Improving Joystick control

### ISSUE THREAD

**Issue by lordfolken** [Openvario/meta-openvario#292]
I don't have a touch screen on my OV. There seem to be still a lot of issues controlling xcsoar with the ov remote. 

* In the menu part, the joystick doesn't work at all. 
* in XCSoar, the airspace warning dialog, is only dismiss-able, but you cannot choose left/right for ignoring the airspace
* using pan mode for selecting things on the map, is very hard to use. Panning is ok, but then the area selected in "what is here" is smaller than the pan steps.

**Comment by linuxianer99** [comment]
Think there is nothing Openvario Image can do on this (expect from the menu which will be integrated in XCSoar) ...

**Comment by lordfolken** [comment]
Uff old. Yes i fixed most of that in xcsoar.

**Comment by linuxianer99** [comment]
Yeah old ... try to close some of the old things to get more overview ;-)
